packages/@ember/-internals/runtime/lib/system/array_proxy.ts

```diff
@@ -218,15 +218,15 @@ class ArrayProxy<T, C extends EmberArray<T> = EmberArray<T>>
     @method replaceContent
     @param {Number} idx The starting index
     @param {Number} amt The number of items to remove from the content.
-    @param {EmberArray} objects Optional array of objects to insert.
+    @param {Array} objects Optional array of objects to insert.
```

Comment:
This can't actually accept an EmberArray.

Reply:
Love that we're fixing docs along the way as part of this. :D